models/comments.models.js

```diff
@@ -0,0 +1,16 @@
+const db = require('../db/connection')
+
+exports.getCommentsByArticleId = (article_id) => {
+	return db
+		.query(
```

Comment:
We need to do a check to see why we're returning empty rows - either an article exists but has no comments, or the article doesn't exist. We want to make sure we're returning the correct status code after all - so you definitely need to have another think about your approach for this task!